### ISSUE TITLE
Add a game to play while someone is afk or something

### ISSUE THREAD

**Issue by Natan515** [lambda-client/lambda-legacy#59]
Like in elytra bot https://github.com/bebeli555/ElytraBot so people can play while they are afk but still want to be safe so they look at there screen. Add Tetris and snake game

**Comment by Avanatiker** [comment]
Maybe..

**Comment by KMatias123** [comment]
u can run 2 programs on ur pc at the same time

**Comment by Natan515** [comment]
Sure, but its less coinvent.

**Comment by Avanatiker** [comment]
Sry not really fitting to the client
